lib/league_stats.rb

```diff
@@ -0,0 +1,134 @@
+require 'csv'
+require 'team'
+require 'game_teams'
+require 'game'
```

Comment:
I don't think this is required, but we can change that when we refactor for inheritance.

Reply:
Thank you for pointing this out